components/atom/tag/src/Standard.js

```diff
@@ -17,11 +17,10 @@ const StandardTag = ({className, closeIcon, icon, label, onClose}) => {
         {label}
       </span>
       {closeIcon && (
-        <span
-          className="sui-AtomTag-secondary-closeable sui-AtomTag-secondary-icon"
-          onClick={handleClick}
-        >
-          {closeIcon}
+        <span className="sui-AtomTag-secondary-closeable" onClick={handleClick}>
```

Comment:
I know this classname was added previously, but I don't understand the concept `secondary` here, is it a modifier or another type of tag? Anyway, classname is not fitting well the SUIT CSS convention, it should be `sui-AtomTagSecondary-closeable`, although for me it would be better have this classnames: `sui-AtomTag-closeable` as parent and then `sui-AtomTag-closeableIcon`.

Reply:
A tag could have 2 icons. Left hand side icon container has "sui-AtomTag-icon" class and right hand side icon container has "sui-AtomTag-secondary-icon" class

Reply:
I have changed "sui-AtomTag-secondary-closeable" to "sui-AtomTag-closeable"

Reply:
Issue created to manage this https://github.com/SUI-Components/sui-components/issues/674